### ISSUE TITLE
Possibility to avoid display comments from loader

### ISSUE THREAD

**Issue by nonameplum** [edeustace/assets-loader#21]
Hi,

Could you add possibility (property etc.) to avoid display comments with requested files by loader like:

```
<!--
      Request:
      --------
...
      Elements raw:
      -----------------
...
      <!-- tags -->
      <!-- Asset Loader -->
...
```

**Comment by edeustace** [comment]
Yes - these are mostly useful in dev mode, will disable them for Prod mode. If one needs to analyse in prod mode, one can use the `logger.assets.loader=TRACE` instead.

**Comment by edeustace** [comment]
@nonameplum try 11.9-SNAPSHOT and if you're happy I'll release 11.9

**Comment by nonameplum** [comment]
Hi, I tried and can say that works great. Thanks.

**Comment by edeustace** [comment]
cool - fix is in 12.0
